models/issues/issue_list.go

```diff
@@ -74,11 +74,11 @@ func (issues IssueList) LoadRepositories(ctx context.Context) (repo_model.Reposi
 
 func (issues IssueList) getPosterIDs() []int64 {
 	return container.FilterSlice(issues, func(issue *Issue) (int64, bool) {
-		return issue.PosterID, true
+		return issue.PosterID, issue.Poster == nil
```

Comment:
I don't agree this change, as function name with `getXXXIDs` will always return all xxx's IDs in other places, but not IDs which is not loaded. Others may confused with it.

Reply:
54e8493e451354e908d86617d190db23896efb14